### DISCUSSION TITLE
Update specific value in field array

### DISCUSSION THREAD

**Discussion by vutpov** [react-hook-form/react-hook-form#2954] [Q&A] [answered]
I want to update the value in the field array using setValue. Increase value by 1 every time the user click the button, but the value doesn't increase bigger than 1.
Here is the codesandbox:
https://codesandbox.io/s/empty-fog-wg85b?file=/src/App.js

**Comment by bluebill1049** [comment, marked as answer]
`const qty = fields[0].value + 1;`

use `getValues()` isntead
